addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -583,6 +583,9 @@ def l10n_in_grouping_key_generator(base_line, tax_values):
                     for gst in ["cgst", "sgst", "igst"]:
                         if any(tag in tags for tag in self.env.ref("l10n_in.tax_tag_%s"%(gst))):
                             line_code = gst
+                        # need to separate rc tax value so it's not pass to other values
+                        if any(tag in tags for tag in self.env.ref("l10n_in.tax_tag_%s_rc" % (gst))):
```

Comment:
Why not tag == self.env.ref... ?  (same above)

Reply:
because maybe multi tag in `tags`
```
>>> a
account.account.tag(135,)
>>> c
account.account.tag(135, 134)
>>> a == c
False
>>> a in c
True
>>> 
```

Reply:
There is only one, but let's do this later.